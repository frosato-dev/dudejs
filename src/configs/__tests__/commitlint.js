const commitLintConfig = require("./commitlint");

it("returns expected commitlint configuration", () => {
  expect(commitLintConfig).toMatchSnapshot();
});
