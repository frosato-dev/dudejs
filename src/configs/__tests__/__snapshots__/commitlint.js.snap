// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns expected commitlint configuration 1`] = `Object {}`;
